source/agora/node/FullNode.d

```diff
@@ -154,7 +154,7 @@ public class FullNode : API
 
     public void start ()
     {
-        this.startPeriodicDiscovery();
+        startPeriodicDiscovery();
```

Comment:
Just missing `this.` and then it's ready to go.

Reply:
The catch is too fast!